ravenframework/Optimizers/RavenSampled.py

```diff
@@ -637,15 +731,22 @@ def _updateSolutionExport(self, traj, rlz, acceptable, rejectReason):
                      'modelRuns': self.counter
                     })
     # optimal point input and output spaces
-    objValue = rlz[self._objectiveVar]
-    if self._minMax == 'max':
-      objValue *= -1
-    toExport[self._objectiveVar] = objValue
+    if len(self._objectiveVar) == 1: # Single Objective Optimization
+      objValue = rlz[self._objectiveVar[0]]
+      if 'max' in self._minMax:
+        objValue *= -1
+      toExport[self._objectiveVar[0]] = objValue
+    else: # Multi Objective Optimization
+      for i in range(len(self._objectiveVar)):
+        objValue = rlz[self._objectiveVar[i]]
+        if self._minMax[i] == 'max':
+          objValue *= -1
+        toExport[self._objectiveVar[i]] = objValue
```

Comment:
This `if ... else ...` can be combined, since `self._objectiveVar` is a list now, and you do not need to treat single and multi-objective separately.

Reply:
Correct! I removed if-else statement.